dataplane/dpdk.cpp

```diff
@@ -0,0 +1,28 @@
+#include <dpdk.h>
+
+namespace dpdk
+{
+
+std::optional<std::string> GetNameByPort(tPortId pid)
+{
+	char cname[256];
```

Comment:
```suggestion
	std::array<char, 256> cname;
```
Wow, found new github feature, didn't know about that